packages/client/src/api/patches.ts

```diff
@@ -90,13 +91,14 @@ export const patchAPI = API => {
     return await enrichRows(rows, tableId)
   }
 
-  // Wipe any HBS formulae from table definitions, as these interfere with
+  // Wipe any HBS formulas from table definitions, as these interfere with
   // handlebars enrichment
   const fetchTableDefinition = API.fetchTableDefinition
   API.fetchTableDefinition = async tableId => {
     const definition = await fetchTableDefinition(tableId)
     Object.keys(definition?.schema || {}).forEach(field => {
       if (definition.schema[field]?.type === "formula") {
+        // @ts-expect-error TODO check what use case removing that would break
         delete definition.schema[field].formula
```

Comment:
Judging by the comment, I'd say this was added because formula column schemas can contain bindings in plaintext - e.g. `{{ user.name }}` etc, and therefore if the table schema was attached to an object which was later fully recursively processed by string-templates, as we do with our component settings, it would end up attempting to processing these bindings inside formula column definitions and fail.

That would end up mutating the definition of the table, which I assume could then be erroneously sent up later and saved.

This was 3 years ago so who knows if that is even still a problem, but that's my guess.